packages/documentation/copy/en/reference/Experimental ESM Support for Node.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Experimental Support for ECMAScript Modules in Node.js
+layout: docs
+permalink: /docs/handbook/experimental-esm-node.html
```

Comment:
we _ideally_ don't want to be changing/breaking URLs in the future 

I can add something like `work-in-progress: true` or `experimental` support to rendering a page tomorrow ( like what we do for deprecated pages https://www.typescriptlang.org/docs/handbook/basic-types.html#string )

Reply:
Yeah, I think that would work great here - thank you!

Reply:
Shipped in https://github.com/microsoft/TypeScript-Website/pull/2123 - if you switch this to its permalink  ( maybe `/docs/handbook/esm-node.html` ?) and add `experimental: true` to this yml and it will be 👍🏻

Reply:
```suggestion
permalink: /docs/handbook/esm-node.html
experimental: true
```